chainer/serializers/npz.py

```diff
@@ -51,25 +51,29 @@ def __call__(self, key, value):
         return ret
 
 
-def save_npz(filename, obj, compression=True):
+def save_npz(file, obj, compression=True):
     """Saves an object to the file in NPZ format.
 
     This is a short-cut function to save only one object into an NPZ file.
 
     Args:
-        filename (str): Target file name.
+        file (str or file-like): Target file to write to.
         obj: Object to be serialized. It must support serialization protocol.
         compression (bool): If ``True``, compression in the resulting zip file
             is enabled.
 
+    .. seealso::
+        :func:`chainer.serializers.load_npz`
+
     """
     s = DictionarySerializer()
     s.save(obj)
-    with open(filename, 'wb') as f:
-        if compression:
-            numpy.savez_compressed(f, **s.target)
-        else:
-            numpy.savez(f, **s.target)
+    if isinstance(file, str):
```

Comment:
In Python 2.x, `unicode` values may be specified as `file`.
I think using `six.string_types` instead of `str` is better here.

Reply:
Fixed!